### PR TITLE
Print process variables with correlation id

### PR DESCRIPTION
**Pull request by boyan-velinov**

### CODE REVIEW

File: multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java [hunk 1]
```diff
@@ -71,7 +73,8 @@ protected void notifyInternal(DelegateExecution execution) {
 
     private void publishDynatraceEvent(DelegateExecution execution, ProcessType processType) {
         DynatraceProcessEvent startEvent = ImmutableDynatraceProcessEvent.builder()
-                                                                         .processId(VariableHandling.get(execution, Variables.CORRELATION_ID))
+                                                                         .processId(VariableHandling.get(execution,
```
Comment (theghost5800) [inline multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java:76]: Extract correlation id into variable and use it at lines 76 and 106

File: multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java [hunk 2]
```diff
@@ -100,7 +103,10 @@ private void logProcessVariables(DelegateExecution execution, ProcessType proces
         getStepLogger().debug(Messages.CLIENT_SPACE, VariableHandling.get(execution, Variables.SPACE_NAME));
         getStepLogger().debug(Messages.CLIENT_ORGANIZATION, VariableHandling.get(execution, Variables.ORGANIZATION_NAME));
         Map<String, Object> processVariables = findProcessVariables(execution, processType);
-        getStepLogger().infoWithoutProgressMessage(Messages.PROCESS_VARIABLES, SecureSerialization.toJson(processVariables));
+        LoggingUtil.logWithCorrelationId(VariableHandling.get(execution, Variables.CORRELATION_ID),
+                                         () -> getStepLogger().infoWithoutProgressMessage(Messages.PROCESS_VARIABLES,
+                                                                                          JsonUtil.toJson(processVariables)));
```
Comment (theghost5800) [inline multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java:108]: Why `SecureSerialization` was not used?

File: multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java [hunk 3]
```diff
@@ -100,7 +103,10 @@ private void logProcessVariables(DelegateExecution execution, ProcessType proces
...
+        // getStepLogger().infoWithoutProgressMessage(Messages.PROCESS_VARIABLES, SecureSerialization.toJson(processVariables));
```
Comment (theghost5800) [inline multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java:109]: remove this

Comment (boyan-velinov) [reply multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java:109]: sure

File: multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java (hunk 1 above)
Comment (boyan-velinov) [reply multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/listeners/StartProcessListener.java:76]: sure

**Review by theghost5800 — APPROVED** [review]

**Pull request merged**